app/src/main/java/nerd/tuxmobil/fahrplan/congress/settings/SettingsScreen.kt

```diff
@@ -0,0 +1,20 @@
+package nerd.tuxmobil.fahrplan.congress.settings
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.platform.LocalContext
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
+
+@Composable
+internal fun SettingsScreen(
+    viewModel: SettingsViewModel = viewModel(
```

Comment:
None of the other screens exposes the viewmodel to the Compose function. I saw this in other projects but never saw a reason to do so. Why would we need to do this here, instead of the pattern of other screens?